app/models/project.rb

```diff
@@ -60,4 +69,15 @@ def related_projects
   def tasks_url
     "#{github_url}/issues"
   end
+
+  def self.parse_git_url(url)
+    url.gsub(/^(((https|http|git)?:\/\/(www\.)?)|git@)github.com(:|\/)/i, '')
+       .gsub(/(\.git|\/)$/i, '')
+  end
+
+  private
+
+  def set_github_repo
+    github_repo = Project.parse_git_url(submitted_github_url).split('/').last
```

Comment:
Useless assignment to variable - `github_repo`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.